pkg/log/log.go

```diff
@@ -43,6 +43,13 @@ func New(out io.Writer, prefix string, debug bool) *Logger {
 // SetDebug sets the debug flag to the value of b
 func (l *Logger) SetDebug(b bool) { l.debug = b }
 
+// SetOutput sets the output destination for Logger.
+// FIXME: We can get rid of this once rkt drops support for 1.4.
+func (l *Logger) SetOutput(w io.Writer) {
+	// No SetOutput exposed in go 1.4.
+	l = New(w, l.Prefix(), l.debug)
```

Comment:
You need to replace the dereferenced value here: http://play.golang.org/p/2xfKAsV5v6